packages/block-editor/src/components/url-input/index.js

```diff
@@ -540,8 +540,7 @@ class URLInput extends Component {
 				>
 					{ suggestions.map( ( suggestion, index ) => (
 						<Button
-							// TODO: Switch to `true` (40px size) if possible
-							__next40pxDefaultSize={ false }
+							__next40pxDefaultSize
```

Comment:
Insert a Paragraph block, select a string range and click the Link button in the block toolbar. I'm not sure what conditions are needed to reach this code naturally — it's not what you normally see in a URLInput in Gutenberg. To force the condition, you can comment out the block with the `if ( isFunction( renderSuggestions ) )` conditional, a few lines above this.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/e215ed55-8a0d-4a2e-bbe2-78b493cec2a6" alt="URLInput, before" width="378">|<img src="https://github.com/user-attachments/assets/d379d47e-f657-4e89-94d7-24adda0fbacd" alt="URLInput, after" width="379">|